cfme/tests/infrastructure/test_customization_template.py

```diff
@@ -21,10 +21,10 @@ def test_customization_template_crud(collection):
                                       image_type='RHEL-6',
                                       script_type='Kickstart',
                                       script_data='Testing the script')
-
+    copy_template_crud = collection.copy(template_crud)
```

Comment:
**REQUIRED** do not do like this - don't add everything inside one test_crud - add separate test for customization template copy / delete multiple etc